themes.js

```diff
@@ -41,7 +42,7 @@ const SEL_DECLARATION = '.dec, .var'; // declaration (like doctype), variable na
 export function setTheme(theme) {
   let {bgColor, textColor, punctuationColor, stringAndValueColor,
        keywordTagColor, commentColor, typeColor, numberColor,
-       declarationColor} = theme;
+       declarationColor, lineHeight} = theme;
 
   let css = `
```

Comment:
Note that we could just add `line-height: ${lineHeight}` here to achieve the desired visual effect. Sadly, that way the lin-height doesn't seem to be respected when copy-and-pasting to Google Slides. Setting it to a specific `px` value instead (like the code already did) seems to work better.